coffee-rest/src/test/java/hu/icellmobilsoft/coffee/rest/log/RequestResponseLoggerTest.java

```diff
@@ -44,42 +45,59 @@
 
     @Test
     @DisplayName("Test printEntity with Json object and application/json;charset=UTF-8 media type")
-    public void printJsonEntityTest() {
+    void printJsonEntityTest() {
         BaseResponse response = new BaseResponse().withMessage("msg").withFuncCode(FunctionCodeType.OK)
                 .withContext(new ContextType().withRequestId("TEST").withTimestamp(OffsetDateTime.now()));
-        MediaType mediaType = new MediaType("application","json",StandardCharsets.UTF_8.displayName());
+        MediaType mediaType = new MediaType("application", "json", StandardCharsets.UTF_8.displayName());
         String responseText = requestResponseLogger.printEntity(response, null, "", false, mediaType);
         Assertions.assertTrue(responseText.startsWith("entity: [{\"context\":{\"requestId\":\"TEST\",\"timestamp\""));
 
     }
 
     @Test
     @DisplayName("Test printEntity with String object and without media type")
-    public void printStringEntityTest() {
-        String responseText = requestResponseLogger.printEntity("<div>Hello World</div>", null, "", false, null);
-        Assertions.assertTrue(responseText.startsWith("entity: [<div>Hello World</div>]"));
+    void printStringEntityTest() {
+        String input = "<div>Hello World</div>";
```

Comment:
esetleg input String kiszervezhető osztály szintre

Reply:
Igen, gondoltam ra, csak mivel ez egy teszt igy olvashatobb, valszeg majd ugy is refaktoraljuk valami egyseges tesztekre